app/build.gradle

```diff
@@ -140,7 +140,6 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.6'
 
     // Firebase
-    implementation 'com.google.firebase:firebase-analytics:17.2.3'
```

Comment:
Nepřidává tu dependency jiné knihovny nebo nepotřebují jí? Vím že Analytics je core pro ostatní Firebase služby. Otestoval jsi Crashlytics? Chtělo by to na test screenu dát asi tlačítko na umělý crash. Nestačilo by jenom v kódu `setAnalyticsCollectionEnabled(false);`?

Reply:
Good point. Crashlytics rika jen recommended, ale to asi jen proto, ze se pak daji ty crashe s tema eventama v analytics nejak spojovat. 
Vypnout to v kodu by asi stacilo, da se to zakazat i nejakym tagem v manifestu, coz je asi jeste lepsi, ale ve chvili kdy to tady nechceme, tak bych to dal proste pryc. 
Podle `./gradlew app:dependencies` to zadna zavislost nepridava. Appka se normalne zkompiluje a spusti. Imho je safe a zadouci to odstranit uplne

Reply:
OK, ale je to fakt škoda. Přijdeme o spoustu užitečných dat a možností :(